pkgs/Canop/dysk/registry.yaml

```diff
@@ -0,0 +1,14 @@
+packages:
+  - type: http
+    repo_owner: Canop
+    repo_name: dysk
+    description: A linux utility to get information on filesystems, like df but better
+    link: https://dystroy.org/dysk/
+    url: https://dystroy.org/dysk/download/{{.Arch}}-{{.OS}}/dysk
```

Comment:
This supports only the latest version.

https://dystroy.org/dysk/install/

> You may download previous releases on [GitHub releases](https://github.com/Canop/dysk/releases).

Reply:
A zip file in GitHub Releases contains all target binaries, so it will be overhead...